docs/sphinx/developers/format-documentation.txt

```diff
@@ -0,0 +1,107 @@
+Adding format/reader documentation pages
+========================================
+
+.. _Velocity: http://velocity.apache.org/
+
+Most documentation pages for the supported formats and readers are
+auto-generated. These pages should not be modified directly. This page
+explains how to amend/extend this part of the Bio-Formats documentation.
+
+The :sourcedir:`Bio-Formats testing framework <components/autogen>` component
+contains most of the infrastructure to run automated tests against the data
+repository.
+
+Formats
+-------
+
+After checking out source code and building all the JAR files (see
+:doc:`building-bioformats`), the supported formats pages can be generated
+using the :program:`ant` ``gen-format-pages`` target under the :file:`autogen`
+component::
+
+  $ ant -f components/autogen/build.xml gen-format-pages
+
+This target will read the metadata for each format stored under
+:source:`format-pages.txt <components/autogen/src/format-pages.txt>` and
+generate a reStructuredText file for each format stored under
+:file:`formats/<formatname>.txt` as well as an index page for all supported
+formats using Velocity_.
+
+The :file:`format-pages.txt` is an INI file where each section corresponds to
+a particular format given by the section header. Multiple key/values should be 
+defined for each section:
```

Comment:
Could you use glossary mark-up or something here? It could do with the terms being in bold if you are going to format it like this really.